tests/gccgo/task.yaml

```
@@ -0,0 +1,14 @@
+summary: Check that snapd builds with gccgo
+prepare: |
+    echo Installing gccgo-6 and pretending it is the default go
+    sudo apt install -y gccgo-6
+    sudo ln -s /usr/bin/go-6 /usr/local/bin/go
+restore: |
+    rm -f /usr/local/bin/go
```

Just an ignorable nitpick, I would drop the `-f` here, so that it fails loudly